src/LoginWithAmazon.js

```diff
@@ -1,18 +1,18 @@
-import React from 'react';
```

Comment:
Noise like this should be eliminated from CRs so you don't distract the reviewer. You can always post a separate CR with just formatting changes.
BTW, using VS Code here will ease the pain because this project is checked in with some VS Code settings.